server/src/test/java/org/opensearch/cluster/coordination/FollowersCheckerTests.java

```diff
@@ -787,4 +797,28 @@ public TransportResponse.Empty read(StreamInput in) {
         }
 
     }
+
+    public void testFollowerCheckTimeoutValue() {
+        Setting<TimeValue> setting1 = LEADER_CHECK_TIMEOUT_SETTING;
+        Settings timeSettings1 = Settings.builder().put(setting1.getKey(), "20s").build();
+
+        ClusterUpdateSettingsResponse response = client().admin()
+            .cluster()
+            .prepareUpdateSettings()
+            .setPersistentSettings(timeSettings1)
+            .execute()
+            .actionGet();
+
+        assertAcked(response);
+        assertEquals(timeValueSeconds(20), setting1.get(response.getPersistentSettings()));
+
+        // cleanup
```

Comment:
> Can we have the below cleanup in finally block and update settings in try block?

Sure, I will add that.

> Also, how about having some assert statements after the cleanup just to be sure?

IMO, assertion after cleanup is not required as the test suite checks for remaining settings and fails the test otherwise